index.js

```diff
@@ -50,10 +50,13 @@ function processProperties (schema, rootSchema, nested, options = {}) {
   return text
 }
 
-function writeDescription (schema, suffix = 'object') {
-  let text = schema.description || `Represents a ${schema.id} ${suffix}`
-  text += `\n  * @name ${upperFirst(schema.id)}`
-  return `  * ${text}\n  *\n`
+function writeDescription (schema) {
```

Comment:
I believe I added suffix for flexibility ie. saying everything is an object can sound redundant.
Prob can be reintroduced in the future through options.